src/main/java/tatracker/commons/core/Messages.java

```diff
@@ -11,7 +11,7 @@
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format!\n%1$s";
     public static final String MESSAGE_INVALID_COMMAND = "Invalid command format!\n\n";
 
-    public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
+    public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command\n\n";
```

Comment:
why are there two line breaks? :O Was that intentional?

Reply:
Yep! Its for joining the messages.
I'm assuming that you always have two line breaks when concatenating another message.